<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>739</width>
    <height>441</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>739</width>
    <height>441</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>739</width>
    <height>441</height>
   </size>
  </property>
  <property name="mouseTracking">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>Group By</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>20</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">text-align: center;
font: 75 16pt &quot;MS Shell Dlg 2&quot;;
color: black;
</string>
   </property>
   <property name="text">
    <string>Group by</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>50</y>
     <width>361</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">text-align: center;
font: 75 12pt &quot;MS Shell Dlg 2&quot;;
color: black;
</string>
   </property>
   <property name="text">
    <string>Double-click to change the properties of the table:</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="GBTable">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>80</y>
     <width>391</width>
     <height>321</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>70</y>
     <width>361</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">text-align: center;
font: 75 12pt &quot;MS Shell Dlg 2&quot;;
color: black;
</string>
   </property>
   <property name="text">
    <string>Group by</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>270</y>
     <width>361</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">text-align: center;
font: 75 12pt &quot;MS Shell Dlg 2&quot;;
color: black;
</string>
   </property>
   <property name="text">
    <string>New column name</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="GBNewColumnName">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>300</y>
     <width>211</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>210</y>
     <width>361</width>
     <height>31</height>
    </rect>
   </property>
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">text-align: center;
font: 75 12pt &quot;MS Shell Dlg 2&quot;;
color: black;
</string>
   </property>
   <property name="text">
    <string>Operation</string>
   </property>
  </widget>
  <widget class="QComboBox" name="GBOperation">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>240</y>
     <width>209</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="GBAddAttribute">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>80</y>
     <width>21</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../../../../Downloads/plus.png</normaloff>../../../../Downloads/plus.png</iconset>
   </property>
  </widget>
  <widget class="QScrollArea" name="GBAttributeScroll">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>110</y>
     <width>241</width>
     <height>81</height>
    </rect>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>239</width>
      <height>79</height>
     </rect>
    </property>
    <widget class="QComboBox" name="GBAttribute">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>181</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="GBAttributeRemove">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>0</y>
       <width>21</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../../../../Downloads/minus.png</normaloff>../../../../Downloads/minus.png</iconset>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>370</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QPushButton" name="GBOkButton">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255)</string>
      </property>
      <property name="text">
       <string>Confirm</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="GBExitButton">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255)</string>
      </property>
      <property name="text">
       <string>Exit</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <action name="actionAdd">
   <property name="icon">
    <iconset>
     <normaloff>../../../../Downloads/plus.png</normaloff>../../../../Downloads/plus.png</iconset>
   </property>
   <property name="text">
    <string>Add</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
